modules/recast/navigation_mesh_editor_plugin.cpp

```diff
@@ -103,24 +103,24 @@ void NavigationMeshEditor::_bind_methods() {
 NavigationMeshEditor::NavigationMeshEditor() {
 
 	bake_hbox = memnew(HBoxContainer);
+
 	button_bake = memnew(ToolButton);
-	button_bake->set_text(TTR("Bake!"));
+	bake_hbox->add_child(button_bake);
 	button_bake->set_toggle_mode(true);
-	button_reset = memnew(Button);
-	button_bake->set_tooltip(TTR("Bake the navigation mesh.") + "\n");
+	button_bake->set_text(TTR("Bake Navigation Mesh"));
 
-	bake_info = memnew(Label);
-	bake_hbox->add_child(button_bake);
+	button_reset = memnew(ToolButton);
 	bake_hbox->add_child(button_reset);
+	button_reset->set_tooltip(TTR("Clear the navigation mesh."));
```

Comment:
This button with text had me pause for a second (I had to check it in editor to understand why it has no text). Maybe add a comment like `// No text, we only use a revert icon set when entering tree`

Reply:
Done.